### PR TITLE
Model tests

### PR DESCRIPTION
**Pull request by ianlofs**
This adds a travis config as well as tests for the models.

### CODE REVIEW

File: seadssite/models/test_models.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+from django.test import TestCase
+from seadssite.models.models import User
+from seadssite.models.models import Device
+from django.core.exceptions import FieldError
+
+
+
+class DeviceTestCase(TestCase):
+    @classmethod
+    def setUpTestData(cls):
```
Comment (nsokolnikov) [inline seadssite/models/test_models.py]: What's this here? How does that work?

File: seadssite/models/test_models.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            print(e)
+
+    def test_register_device_raises_device_registered_by_different_user(self):
+        try:
+            Device.objects.register_device(1, 'test_device1', self.user2)
+            self.assertRaisesMessage(FieldError, 'This device has already been registered to a different user.')
+        except FieldError as e:
+            print(e)
+
+    '''raise FieldError('This device has already been disactivated.', self)'''
```
Comment (nsokolnikov) [inline seadssite/models/test_models.py]: Remove this line. Dead code.

File: seadssite/models/test_models.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            self.device3.deactivate_device()
+            self.device3.deactivate_device()
+            self.assertRaisesMessage(FieldError, 'This device has already been disactivated.')
+        except FieldError as e:
+            print(e)
+
+    def test_reactivate_device(self):
+        self.device4.reactivate_device()
+        device = Device.objects.get(device_id=self.device4.device_id)
+        self.assertEqual(self.device4.is_active, True)
```
Comment (nsokolnikov) [inline seadssite/models/test_models.py]: Reactivate device should attach the device to a user right away. Those two things need to be coupled.

**Comment by nsokolnikov** [comment]
Once we fix:
- [x] coupling reactivating device to attaching it to a user
- [x] coupling deactivating device to removing it from the user it's registered to

then it's okay to merge.

**Comment by ianlofs** [comment]
@nsokolnikov how does this look?

**Pull request merged**